examples/hosting-attachedDomain/api_attach_domain_to_web_hosting.md

```diff
@@ -0,0 +1,89 @@
+How to attach domains to web hosting offer using php wrapper?
+-------------------------------------------------------------
+
+This documentation will help you to create, edit and delete attached domains to your webhosting offer. This documentation is the equivalent of [MultiDomain SoAPI](http://www.ovh.com/soapi/en/?method=multiDomainAdd) and [SubDomain SoAPI](http://www.ovh.com/soapi/en/?method=subDomainAdd)
+
+## Compatibility note
+
+MultiDomains and SubDomains options are merged into one option called **Attached Domains**. You can set up subdomains and other domains to your web hosting with this new option.
+
+## Requirements
+
+- Having PHP 5.2+
+- Having a domain at OVH
+- Having an hosting account
+
+## Download PHP wrapper
+
+- Download the latest release **with dependencies** on github: https://github.com/ovh/php-ovh/releases
+
+```bash
+# When this article is written, latest version is 2.0.0
+wget https://github.com/ovh/php-ovh/releases/download/v2.0.0/php-ovh-2.0.0-with-dependencies.tar.gz
+```
+
+- Extract it into a folder
+
+```bash
+tar xzvf php-ovh-2.0.0-with-dependencies.tar.gz 
+```
+
+## Create a new token
+You can create a new token using [this url](https://api.ovh.com/createToken/?GET=/hosting/web/my_domain/attachedDomain&POST=/hosting/web/my_domain/attachedDomain&GET=/hosting/web/my_domain/attachedDomain/*&PUT=/hosting/web/my_domain/attachedDomain/*&DELETE=/hosting/web/my_domain/attachedDomain/*). Keep application key, application secret and consumer key to complete the script.
+
+Be warn, this token is only validated for this script and for hosting called **my_domain**. Please replace **my_domain** by your web hosting reference!
+If you need a more generic token, you had to change right field.
```

Comment:
s/you had to change right field/make sure to set the rights field to your needs/